python/foglamp/services/core/api/support.py

```diff
@@ -89,6 +91,42 @@
     return web.json_response({"bundle created": bundle_name})
 
 
+async def get_syslog_entries(request):
+    """ Returns a list of syslog trail entries sorted with most recent first and total count
+        (including the criteria search if applied)
```

Comment:
default should return  latest 20 enteries

Reply:
already there